src/components/product-detail/product-detail.component.html

```diff
@@ -0,0 +1,31 @@
+<div *ngIf="product" class="container-md mt-4">
+    <div class="row">
+        <div class="col">
+            <img class="img-fluid" [src]="product.imageURL" />
+        </div>
+        <div class="col">
+            <div class="card h-100">
+                <div class="card-header fs-1">
+                    <h1>{{ product.productName }}</h1>
+                </div>
+                <div class="card-body">
+                    <h2 class="card-title">
+                        {{ product.productPrice | currency }}
+                    </h2>
+                    <p class="card-text">
+                        Shipping: {{ product.shippingPrice | currency }}
```

Comment:
Same inquiry as before. I just wonder if we need a shipping/additional fees ON a product. Maybe we replace it with something like quantityInStock, weight, dimensions, so on so forth. We could just keep it like this for now though its not a huge deal.